content/en/shader-editor/window-layout/assets-pane.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Assets Pane
+template: shader-editor-page.tmpl.html
+position: 3
+---
+
+The Assets Pane is where the project’s graphs, materials and textures appear.
+
+![Assets Pane][1]
+
+| How to | |
+|---|---|
+| Select an item | Click an item and see it's properties in the [Inspector Pane][2]. |
+| Create a new graph | Enter a graph name in the 'graph name' entry and either press enter or click '+ NEW'. |
+| Delete an item | Select the item and press the delete button or access the item's context menu and select 'Delete'. |
+| Filter the list of items | Enter text into the filter entry box. |
```

Comment:
Nice to have: images/gifs of each option

Reply:
yup, followup pr?

Reply:
Yeo